### PR TITLE
[INJIMOB-956]: Mock server to host svg template.

### PR DESCRIPTION
**Pull request by balachandarg-tw**
Mock server to host svg template locally.

### CODE REVIEW

File: inji-vc-renderer-mock-service/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const app = express();
+const port = 3000;
```
Comment (swatigoel) [inline inji-vc-renderer-mock-service/index.js]: don't use 3000 as it's very common for UI components

File: inji-vc-renderer-mock-service/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "sample-server",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "mock-renderer": "node index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (swatigoel) [inline inji-vc-renderer-mock-service/package.json]: what is this license?

**Review by swatigoel — CHANGES_REQUESTED** [review]

File: inji-vc-renderer-mock-service/package.json (hunk 2 above)
Comment (balachandarg-tw) [reply inji-vc-renderer-mock-service/package.json]: ISC is the default license added during project creation, updated to mpl2.0

File: inji-vc-renderer-mock-service/index.js (hunk 1 above)
Comment (balachandarg-tw) [reply inji-vc-renderer-mock-service/index.js]: Updated to 3008

**Review by swatigoel — APPROVED** [review]

**Pull request merged**